src/main/java/dao/Reader.java

```diff
@@ -0,0 +1,7 @@
+package dao;
+
+import java.util.List;
+
+public interface Reader {
+    List<String> reader(String fileName);
```

Comment:
```suggestion
    List<String> read(String fileName);
```
Verbs are usually used for method naming